server/models/Request.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user_id: {
```

Comment:
It would be ideal to subscribe to the `camelCase` convention of JS for identifiers/properties so let's rename these fields 👍🏼

Reply:
switched to camelCase